live-build/misc/upgrade-scripts/verify

```diff
@@ -41,6 +41,20 @@ function cleanup() {
 	#
 	[[ -z "$CONTAINER" ]] && return
 
+	#
+	# We need to collect logs within the container and move to
+	# container's host otherwise those will delete when container
+	# is deleted. We leverage support_info script to do the collection
+	# and only collect "log" and "dropbox" types. User who wish to add
+	# more collection types from support_info here should check if
+	# they work in the container.
+	# We collect only one copy to limit space usage.
+	#
+	report_progress 70 "Collecting verification logs"
+	rm -rf "${UPDATE_VERIFY_LOG_DIR:?}/*"
```

Comment:
Why not remove the entire directory with:
```
rm -rf "${UPDATE_VERIFY_LOG_DIR:?}"
```
?

Does the `/*` part buy us anything? It means dot-files will not be removed, but idk if that's a bug, or a feature?